docs/ecosystem.md

```diff
@@ -0,0 +1,57 @@
+The following document provides an ecosystem overview of how the various tools compatible with Cirq can work together to enable quantum computing research.
+
+![image alt text](images/ecosystem.png )
+
+* **Research Libraries and Tools:** Libraries and tools used for researching new quantum algorithms and designing and preparing experiments on quantum devices
+
+* **Cirq**: A framework specifically for programming noisy intermediate quantum computers
+
+* **Quantum Cloud Services:** Cirq can connect to a variety of quantum cloud services. Behind each cloud service, one can invoke either an actual quantum computer or a quantum circuit simulator
```

Comment:
Instead of "actual quantum computer", would "physical hardware device" or something similar be more accurate?

Reply:
"Run quantum algorithms on a quantum processor or a simulator."?